server/planning/planning.py

```diff
@@ -69,13 +80,158 @@ class PlanningService(superdesk.Service):
     },
 
     # Planning Details
-    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16.5.1
+    # NewsML-G2 Event properties See IPTC-G2-Implementation_Guide 16
     # probably can skip this subsection, although its documented in iptc impl guide this way
-    'planning_details': {
+    'planning_item': {
         'type': 'dict',
         'schema': {
-            # TODO: fill in from NewsML-G2 version 2.21 spec
-        }
+            # See IPTC-G2-Implementation_Guide 16.1
+            'item_meta': {
+                'type': 'dict',
+                'schema': {
+                    'item_class': {'type': 'string'}
+                }
+            },
+            # See IPTC-G2-Implementation_Guide 16.2
+            'content_meta': {
+                'type': 'dict',
+                'schema': {
+                    'qcode': {'type': 'string'},
+                    'name': {'type': 'string'}
+                }
+            },
+            # See IPTC-G2-Implementation_Guide 16.3
+            'news_coverage_set': {
```

Comment:
yeah, @vied12 were discussing this yesterday and I thought maybe it was better nested because I couldn't think of a case where we would want the coverage WITHOUT the planning information (especially since the item/content metadata is on the planning item, not the coverage), and thought this way saves a lookup in most cases.  what do you think?